node-commons/pom.xml

```diff
@@ -51,6 +51,16 @@
             <artifactId>guava</artifactId>
             <version>[${guava.version},)</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>iotdb-thrift-commons</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
```

Comment:
It looks like we ended up making the Node-common module rely on almost all of the thrift classes... @JackieTien97 What do you think?

Reply:
Fixed~